actors/builtin/storage_power/storage_power_actor.go

```diff
@@ -294,32 +295,30 @@ func (a *StoragePowerActor) OnMinerEnrollCronEvent(rt Runtime, eventEpoch abi.Ch
 	return &adt.EmptyValue{}
 }
 
-func (a *StoragePowerActor) ReportConsensusFault(rt Runtime, blockHeader1, blockHeader2 []byte, slashee addr.Address, faultEpoch abi.ChainEpoch, faultType ConsensusFaultType) *adt.EmptyValue {
+func (a *StoragePowerActor) ReportConsensusFault(rt Runtime, blockHeader1, blockHeader2 []byte, target addr.Address, faultEpoch abi.ChainEpoch, faultType ConsensusFaultType) *adt.EmptyValue {
 	// TODO: determine how to reward multiple reporters of the same fault within a single epoch.
 
 	isValidConsensusFault := rt.Syscalls().VerifyConsensusFault(blockHeader1, blockHeader2)
 	if !isValidConsensusFault {
 		rt.Abort(exitcode.ErrIllegalArgument, "spa.ReportConsensusFault: unverified consensus fault")
 	}
 
-	slasherAddr := rt.ImmediateCaller()
-	var amountToSlasher abi.TokenAmount
+	reporter := rt.ImmediateCaller()
 	var st StoragePowerActorState
-	rt.State().Transaction(&st, func() interface{} {
-		claimedPower, powerOk, err := getStoragePower(adt.AsStore(rt), st.ClaimedPower, slashee)
+	reward := rt.State().Transaction(&st, func() interface{} {
+		store := adt.AsStore(rt)
+		claimedPower, powerOk, err := getStoragePower(store, st.ClaimedPower, target)
 		if err != nil {
 			rt.Abort(exitcode.ErrIllegalState, "spa.ReportConsensusFault failed to read claimed power for fault: %v", err)
 		}
 		if !powerOk {
 			rt.Abort(exitcode.ErrIllegalArgument, "spa.ReportConsensusFault: miner already slashed")
 		}
-		Assert(claimedPower.GreaterThan(big.Zero()))
+		Assert(claimedPower.GreaterThanEqual(big.Zero()))
 
-		currPledge, pledgeOk := st.getMinerPledge(slashee)
-		if !pledgeOk {
-			rt.Abort(exitcode.ErrIllegalArgument, "spa.ReportConsensusFault: miner has no pledge")
-		}
-		Assert(currPledge.GreaterThan(big.Zero()))
+		currPledge, err := st.getMinerPledge(store, target)
+		abortIfError(rt, err, "failed to get miner pledge")
+		Assert(currPledge.GreaterThanEqual(big.Zero()))
```

Comment:
This changed too, it assumed the pledge function is always non-zero, but doesn't need that assumption to be correct.